### PR TITLE
Lite: Split_V test cases improvised

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
1:> Test cases improvised to support 2 more datatypes.
2:> Error message improvised.

### CODE REVIEW

**Review by shahzadlone — DISMISSED** [review]

File: tensorflow/lite/kernels/split_v_test.cc [hunk 1]
```diff
@@ -12,8 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#include <initializer_list>
```
Comment (haozha111) [inline tensorflow/lite/kernels/split_v_test.cc]: I think the previous ordering is better.

File: tensorflow/lite/kernels/split_v_test.cc [hunk 2]
```diff
@@ -112,57 +111,93 @@ TEST(SplitVOpTest, TwoDimensional) {
...
+}
+
+TEST(SplitVOpTest, TwoDimensionalUint8) {
+  // Input shape: {4, 3}
+  // size_splits: {1, 1, 3}
+  // axis: 0
+  // We should have 3 outpus with shapes respectively:
+  //  output 0 : {1, 3}
+  //  output 1 : {1, 3}
+  //  output 1 : {2, 3}-t
```
Comment (haozha111) [inline tensorflow/lite/kernels/split_v_test.cc]: remove?

File: tensorflow/lite/kernels/split_v_test.cc [hunk 3]
```diff
@@ -112,57 +111,93 @@ TEST(SplitVOpTest, TwoDimensional) {
...
+}
+
+TEST(SplitVOpTest, TwoDimensionalInt16) {
+  // Input shape: {4, 3}
+  // size_splits: {1, 1, 3}
+  // axis: 0
+  // We should have 3 outpus with shapes respectively:
+  //  output 0 : {1, 3}
+  //  output 1 : {1, 3}
+  //  output 1 : {2, 3}-t
```
Comment (haozha111) [inline tensorflow/lite/kernels/split_v_test.cc]: remove?

File: tensorflow/lite/kernels/split_v_test.cc (hunk 1 above)
Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/split_v_test.cc]: @haozha111 : This was reported in static check, so modified, still need to revert?

File: tensorflow/lite/kernels/split_v_test.cc (hunk 2 above)
Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/split_v_test.cc]: No removed, just reordered, it is the same, no change.

File: tensorflow/lite/kernels/split_v_test.cc (hunk 3 above)
Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/split_v_test.cc]: same here also, not removed, it is the same.

File: tensorflow/lite/kernels/split_v_test.cc (hunk 2 above)
Comment (haozha111) [reply tensorflow/lite/kernels/split_v_test.cc]: Sorry for confusion. I mean can we remove the '-t' at the line end? What's it for?

File: tensorflow/lite/kernels/split_v_test.cc (hunk 3 above)
Comment (haozha111) [reply tensorflow/lite/kernels/split_v_test.cc]: '-t' seems weird.

File: tensorflow/lite/kernels/split_v_test.cc [hunk 4]
```diff
@@ -112,57 +111,93 @@ TEST(SplitVOpTest, TwoDimensional) {
...
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
+      {
+          {1, 2, 3, 4, 5, 6, 7, 8},
+          {9, 10, 11, 12, 13, 14, 15, 16},
+      });
+}
+
+TEST(SplitVOpTest, TwoDimensionalUint8) {
+  // Input shape: {4, 3}
+  // size_splits: {1, 1, 3}
```
Comment (haozha111) [inline tensorflow/lite/kernels/split_v_test.cc]: I think this should be {1, 1, 2} ?

File: tensorflow/lite/kernels/split_v_test.cc [hunk 5]
```diff
@@ -112,57 +111,93 @@ TEST(SplitVOpTest, TwoDimensional) {
...
+  //  output 1 : {2, 3}-t
+  Check<uint8_t, TensorType_UINT8>(
+      /*axis=*/0, {4, 3}, {3}, {{1, 3}, {1, 3}, {2, 3}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12}, {1, 1, 2},
+      {{1, 2, 3}, {4, 5, 6}, {7, 8, 9, 10, 11, 12}});
+}
+
+TEST(SplitVOpTest, TwoDimensionalInt16) {
+  // Input shape: {4, 3}
+  // size_splits: {1, 1, 3}
```
Comment (haozha111) [inline tensorflow/lite/kernels/split_v_test.cc]: 1,1,2

File: tensorflow/lite/kernels/split_v_test.cc (hunk 2 above)
Comment (haozha111) [reply tensorflow/lite/kernels/split_v_test.cc]: please also correct the comment here:
output 1: xxx
output 2: xxx
output 3: xxx

same as below.

File: tensorflow/lite/kernels/split_v_test.cc (hunk 1 above)
Comment (haozha111) [reply tensorflow/lite/kernels/split_v_test.cc]: Yes, we usually import headers in the following order:
1) C++ STL
2) Google-specific libraries (testing, etc)
3) Everything under tensorflow/

So please revert those. Thanks!

File: tensorflow/lite/kernels/split_v_test.cc (hunk 2 above)
Comment (ANSHUMAN87) [reply tensorflow/lite/kernels/split_v_test.cc]: @haozha111: These are not part of my changes, but sure i will handle it, Thanks!

**Review by haozha111 — APPROVED** [review]
Thanks again for your changes!

**Review by haozha111 — APPROVED** [review]

**Pull request merged**